### ISSUE TITLE
Returning "NaN" values from fetch function

### ISSUE THREAD

**Issue by PubuduSaneth** [mirnylab/cooler#85]
Hi all 

I downloaded "Rao2014-HMEC-MboI-allreps-filtered.1000kb.cool" file from cooler ftp site "ftp://cooler.csail.mit.edu/coolers/hg19/" and executed simple fetch command. 

Fetch command returned "NaN" values for both interacting chromosomal regions and balance column, but a numeric value for count column. I feel that receiving "NaN" for chromosomal regions with non-zero interaction count is strange and would be thankful if someone can help me to understand this result.

My python code is as following
```
cool_file = "/Users/pubudu/Documents/RefData/HiCData/COOLER/Dixon2012-H1hESC-HindIII-allreps-filtered.1000kb.cool"
c = cooler.Cooler(cool_file)
print (c.matrix(as_pixels=True, join=True, balance=True).fetch("chr10:6052652-6104288", "chr10:7052652-8104288"))
```
> output
  chrom1  start1  end1 chrom2  start2  end2  count  balanced
0    NaN     NaN   NaN    NaN     NaN   NaN  11119       NaN
1    NaN     NaN   NaN    NaN     NaN   NaN   1601       NaN

Best

**Comment by nvictus** [comment]
Hi Pubudu,

That's due to a bug with the combination of `as_pixels` and `join` options. Will fix right away, but as an immediate workaround you can simply do the following, which is completely equivalent:

```python
df = c.matrix(as_pixels=True, balance=True).fetch("chr10:6052652-6104288", "chr10:7052652-8104288"))
df = cooler.annotate(df, c.bins()[['chrom', 'start', 'end]])
```
Thanks for catching this!

**Comment by nvictus** [comment]
Fixed by a5c6ef80dc

**Comment by PubuduSaneth** [comment]
Thank you Nezar for giving an alternative way.
All the best...
